support-data-rest/src/main/java/org/uniprot/api/taxonomy/request/TaxonomyRequestDTO.java

```diff
@@ -32,5 +32,5 @@
     private String facets;
 
     @Positive(message = "{search.positive}")
```

Comment:
Doubt: Should we allow -1 here to enable download everything, Can you please look all the other requests about this comment?